### ISSUE TITLE
Session may need access control module

### ISSUE THREAD

**Issue by HarryShaunWang** [source-academy/cadet-frontend#756]
When teacher is sharing his/her code screen to students by Session. He/She may doesn't want students to modify the code on his/her screen. When teachers only want to display or share code, there need be a option for sharer to choose whether he/she want others to modify the code shared. This option should be able to be change at any time the sharer want.

**Comment by jiachen247** [comment]
@HarryShaunWang Wait how does this session sharing happen I hope you arent referring to sharing the JWT but the physical laptop. This sounds more like a feature request. Maybe you can clarify. 

Are you referring to a one way collaborative feature?

**Comment by martin-henz** [comment]
Yes, I think the Issues here can be used for Feature Requests. The labels "Enhancement" and "Proposal" seem appropriate for this.
